.github/workflows/pull-request.yml

```
@@ -462,3 +462,14 @@ jobs:
         run: |
           echo "Error: Pull request target branch must be 'develop-postgres'. Please refer PR_GUIDELINES.md"
           exit 1
+
+  Validate-Coderabbit:
```

actionlint (1.7.4)

"runs-on" section is missing in job "Validate-Coderabbit"

(syntax-check)